guvnor-ala/guvnor-ala-services-api/src/main/java/org/guvnor/ala/services/api/RuntimeQuery.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.guvnor.ala.services.api;
+
+/**
+ * This class models the query parameters for a query against the runtime system.
+ */
+public class RuntimeQuery {
```

Comment:
this is a generic comment because a runtime in this context might be whatever thing provisioned.